src/client/containers/authentification/authentification.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import FormComponent from '../formComponent/formComponent';
+import { formFieldsInfo } from '../../helpers/constants';
+import './authentification.css';
+
+class Authentification extends Component {
+    state = {
+        isMember: true
+    }
+    
+    onLinkClick = () => {
+        this.setState((state)=>({isMember: !state.isMember}));
```

Comment:
I meant the this.setState({ isMember: !this.state.isMember });

Reply:
Done.